lib/OpenQA/WebAPI/Controller/API/V1/Search.pm

```diff
@@ -147,12 +147,22 @@ sub _search_job_templates {
     my ($self, $keywords, $limit) = @_;
 
     my @results;
-    my $last_group = -1;
-    my $like       = {like => "%${keywords}%"};
-    my $templates  = $self->schema->resultset('JobTemplates')->search({-or => {name => $like, description => $like}})
-      ->slice(0, $limit);
+    my $last_group     = -1;
+    my $like           = {like => "%${keywords}%"};
+    my $test_suite_ids = $self->_get_test_suite_ids($keywords);
```

Comment:
Can you clarify why this isn't a separate query? Is it intended to work as a fallback mechanism?